packages/block-editor/src/components/block-list/index.js

```diff
@@ -195,6 +195,10 @@ class BlockList extends Component {
 			className,
 			blockClientIds,
 			rootClientId,
+			moverOptions = {
```

Comment:
I've had feedback on one of my PRs before that it's preferable for component props not to be in an object like this. I think the reason was that memoisation of props is generally only shallow. Just thought I'd pass that along 😄

Reply:
bleh, so more props?